hacks.js

```diff
@@ -0,0 +1,27 @@
+// Addresses an issue where RTC peers are not properly garbage collected in Chromium:
+// https://bugs.chromium.org/p/chromium/issues/detail?id=825576
+// This error leads to WebTorrent Desktop and Chromium-based browsers being completely
+// unable to seed until the process is restarted:
+// https://github.com/webtorrent/webtorrent/issues/1981
+let gcInterval = 1
+function forcePeerGC () {
+  let peer = new RTCPeerConnection()
+  setTimeout(() => {
+    peer.close()
+    peer = null
+  }, 10)
```

Comment:
is there a reason it has to create 5 peers??

Reply:
Probably not. That was in the original code I found.

I'll bet that it was there as a demo; just to create a bunch of fake peers, so that the microtask has something to garbage collect.

I'll do some testing, and remove that part if it isn't necessary. Thanks for the comment.